### ISSUE TITLE
md-select role should be listbox

### ISSUE THREAD

**Issue by calebegg** [angular/material#3859]
md-select has an ARIA role of "combobox", but it's not actually a combobox; it's a listbox. A combobox has a text field for the user to type in new items.

> A presentation of a select; usually similar to a textbox where users can type ahead to select an option, or type to enter arbitrary text as a new item in the list.

-- http://www.w3.org/TR/wai-aria/roles#combobox

A combobox is required to have a child element that has role "textbox".

I think listbox is more appropriate:

> A widget that allows the user to select one or more items from a list of choices.

-- http://www.w3.org/TR/wai-aria/roles#listbox

**Comment by cleever** [comment]
I Agree.

Combobox is appropriate to md-autocomplete.

md-select is other thing, maybe a listbox.

**Comment by marcysutton** [comment]
After talking this over with an accessibility colleague of mine and doing some testing, I have found that the role of `combobox` is actually correct for JAWS and NVDA–that is what is announced for a native select. In VoiceOver on OSX, a native select is announced as "popup button", so the role will have to change depending on the user agent. I would prefer to keep the same role for all user agents, but if there is an easy way to swap out the role with minimal side-effects I will go ahead and do it.

I have opened a separate issue, #4977, to track the remaining issues with the `combobox` role; mainly, that the options are not rendered on page load, throwing an error in multiple accessibility checkers.
